stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -1878,6 +1871,14 @@ impl NakamotoChainState {
             return Err(ChainstateError::InvalidStacksBlock(msg.into()));
         }
 
+        debug!("Process staging Nakamoto block";
+               "consensus_hash" => %next_ready_block.header.consensus_hash,
+               "stacks_block_hash" => %next_ready_block.header.block_hash(),
+               "stacks_block_id" => %next_ready_block.header.block_id(),
+               "burn_block_hash" => %next_ready_block_snapshot.burn_header_hash,
+               "parent_block_id" => %next_ready_block.header.parent_block_id,
+        );
```

Comment:
It's helpful having this appear earlier (where it was before), because there's a bunch of try macros between the original location and where its moved to. Those try macros will materialize a warn log in an invoking function, but there would be no logs indicating which block the node was considering processing when the error occurred.